src/Pages/Homepage.jsx

```diff
@@ -1,71 +1,77 @@
 import axios from "axios";
 import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
 import { Category, Navbar, VideoThumbnail } from "../Components";
-import "./CSS/homepage.css"
+import "./CSS/homepage.css";
 
 export function Homepage() {
+  const [categories, setCategories] = useState([]);
+  const [videos, setVideos] = useState([]);
```

Comment:
You can maintain ‘state’ globally for the videos, categories and such data.

Reply:
rightly pointed out by @MarkVed17